tools/check/check-timeout.go

```diff
@@ -100,6 +100,7 @@ func init() {
 		"testDBSuite1.TestAddIndexWithSplitTable",
 		"testSerialDBSuite.TestAddIndexWithShardRowID",
 		"testColumnTypeChangeSuite.TestColumnTypeChangeFromDateTimeTypeToOthers",
+		"TestMemStoreConcurrent",
```

Comment:
Why do you add this entry?

Reply:
> TestConcurrent takes at least 10s, so change its name to be less general, and add it to timeout whitelist

Reply:
It is take 10s after the migration? I don't see `TestConcurrent` in the whitelist previously.

Reply:
Writer do insertion and deletion for at least 10s.

https://github.com/pingcap/tidb/blob/b0c9d19c5ebe2849a3cf9820cfcbdb3905ad239c/store/mockstore/unistore/lockstore/lockstore_test.go#L171-L175

Before migration this function will be masked as `TestT`

https://github.com/pingcap/tidb/blob/b0c9d19c5ebe2849a3cf9820cfcbdb3905ad239c/store/mockstore/unistore/lockstore/lockstore_test.go#L28-L30

```
OK: 3 passed, 1 skipped
--- PASS: TestT (10.58s)
```

Reply:
If adding another test to whitelist is not that good, maybe we can reduce the time to 3s or less...

Reply:
Never mind. I just want to see whether it is a new issue introduced by this PR.

You can make a follow up to improve it or make an issue first so that we don't lost trace to TestMemStoreConcurrent,